modules/installation-infrastructure-user-infra.adoc

```diff
@@ -74,6 +74,13 @@ endif::ibm-z-kvm[]
 . Ensure that your network infrastructure provides the required network connectivity between the cluster components. See the _Networking requirements for user-provisioned infrastructure_ section for details about the requirements.
 
 . Configure your firewall to enable the ports required for the {product-title} cluster components to communicate. See _Networking requirements for user-provisioned infrastructure_ section for details about the ports that are required.
++
+[IMPORTANT]
+====
+By default, port `1936` is accessible for a {product-title} cluster, because each control plane node needs access to this port. 
+
+Do not use the Ingress load balancer to expose this port, because exposing the port in this way could expose sensitive information, such as statistics and metrics, related to Ingress Controllers.
```

Comment:
```suggestion
Avoid using the Ingress load balancer to expose this port, because doing so might result in the exposure of sensitive information, such as statistics and metrics, related to Ingress Controllers.
```